doc/build-instructions/Build_Instructions_V17.md

```diff
@@ -100,11 +100,11 @@ apt-get update \
     ...
 ```
 
-2. The previous step installed g++-7 and gcc-7 packages, which might be different
+2. The previous step installed g++-10 and gcc-10 packages, which might be different
 than the default version installed on your system. Export variables to set the
 version used in the build.
 ```
-export CC=gcc-7 CXX=g++-7
+export CC=gcc-10 CXX=g++-10
```

Comment:
Ubuntu:18.04 provides gcc-8 and g++8.
Alternatively, we could advise using Ubuntu:20.04 by updating the `mkdocker.sh` commands:
```sh
bash mkdocker.sh ... --version=20 ...
```

Reply:
Updated to Ubuntu 20.